source/stylesheets/_settings.scss

```diff
@@ -1,18 +1,58 @@
 // Breakpoints
-$desktop: new-breakpoint(min-width 768px 12);
-$tablet: new-breakpoint(max-width 767px);
-$mobile: new-breakpoint(max-width 480px);
+$desktop: new-breakpoint(min-width 800px 12);
+$tablet: new-breakpoint(max-width 799px);
+$mobile: new-breakpoint(max-width 520px);
+
 
 // Colors
 $yellow-tribune: #ffc200;
 $teal-tribune: #008990;
 
+$black: #000000;
+$gray-medium: #aaaaaa;
+$gray-light: #ebebeb;
+$white: #ffffff;
+
 $bg-contribute: white;
 $bg-nav: #000000;
 
 $link-nav: white;
 
-// Typography
+
+// Fonts
+$league-gothic: 'league-gothic', sans-serif;
+
 
 // Mixins
+@mixin hide-for-mobile {
+  @include media($mobile) {
+    display: none;
```

Comment:
OK, I updated to use visually hidden for things like the footer so that all links would remain accessible at any size. I also updated to use visually-hidden for the membership benefits grid so that all benefits would be accessible. Also, now all the benefits will just show and not be hidden if somebody doesn't have JS, which is also better.

I left the hide-for-mobile and show-for-mobile to use for cases where content is being displayed differently, but it doesn't affect the content.